src/impy/models/dpmjetIII.py

```diff
@@ -78,24 +78,35 @@ def __init__(self, event_kinematics, seed=None, logfname=None):
         self._max_A2 = event_kinematics.A2
 
         self.event_kinematics = event_kinematics
-        dpm_conf = impy_config["dpmjetIII"]
 
         info(1, "Initializing DPMJET-III")
+
+        if self.version == "3.0-6":
```

Comment:
These should be class variables. So by subclassing one could override these paths. Like that it's impossible to edit the paths without messing with the source code directly.

Reply:
Done